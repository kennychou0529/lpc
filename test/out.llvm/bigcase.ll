
; line 0
;================================================================================
; scope: global scope (level : 0)

; types
%T_pointer_1 = type i8*
%T_text = type __file

; line 2
;================================================================================
; scope: program scope (level : 1)

; types
%T_range_2 = type __range

; program variables
@_input = dso_local global %T_text zeroinitializer
@_output = dso_local global %T_text zeroinitializer
@Count = dso_local global i32 zeroinitializer
@Index = dso_local global %T_range_2 zeroinitializer
