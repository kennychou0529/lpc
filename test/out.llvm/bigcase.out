    1 TRUE is True, and this is dumb.
    2 Small even number.
    3 Boo Boo Boo
    4 Small even number.
    5
    6 Small even number.
    7 Big Number
    8 Big Number
    9 Big Number
   10
